src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/IParameterModelBaseConvention.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationModels
+{
+    /// <summary>
+    /// Allows customization of the <see cref="ParameterModelBase"/>.
+    /// </summary>
+    /// <remarks>
+    /// To use this interface, create an <see cref="System.Attribute"/> class which implements the interface and
+    /// place it on an action method parameter.
```

Comment:
It's worth another paragraph here that explains that this is applies to the various other parameter types.